runtime/src/snapshot_utils.rs

```diff
@@ -623,11 +693,14 @@ pub fn add_bank_snapshot<P: AsRef<Path>>(
     let bank_snapshots_dir = get_bank_snapshots_dir(bank_snapshots_dir, slot);
     fs::create_dir_all(&bank_snapshots_dir)?;
 
-    // the bank snapshot is stored as bank_snapshots_dir/slot/slot
-    let snapshot_bank_file_path = bank_snapshots_dir.join(get_snapshot_file_name(slot));
+    // the bank snapshot is stored as bank_snapshots_dir/slot/slot(SNAPSHOT_BANK_TEMPORARY_FILENAME_EXTENSION)
+    let mut bank_snapshot_path = bank_snapshots_dir.join(get_snapshot_file_name(slot));
+    bank_snapshot_path.set_extension(BANK_SNAPSHOT_PRE_FILENAME_EXTENSION);
```

Comment:
Yeah, I think this would be good. And probably even a single function that takes the `bank_snapshots_dir` and the `slot` and return the paths for _both_ the `Pre` and `Post` bank snapshots, since they are both almost always used.